src/emm/services/modules_service.py

```diff
@@ -225,9 +230,99 @@ def git_commit_modules(self, commit: str) -> None:
 
         :param commit: Commit content for all changes.
         """
-        enabled_modules = self.get_all_modules(True)
         self.git_service.commit_all(commit)
+        enabled_modules = self.get_all_modules(True)
         for module in enabled_modules:
             module_data = self.get_module_data(module)
             module_location = Path(module_data.prefix) / module
             self.git_service.commit_all(commit, module_location)
+
+    def add_pr_links(self, comments: Dict[str, str]) -> List[str]:
```

Comment:
There is a design principle called [command/query separation](https://en.wikipedia.org/wiki/Command%E2%80%93query_separation) that might make this code simpler and easier to use. The basic idea that is function should perform an action or query data, but not both. 

In this case, by separating the code that creates the links for the PRs from the code that updates the PR with the link would make the code easier to follow and test.